server/cdc/publisher.go

```diff
@@ -63,13 +64,17 @@ func NewPublisher(dbName string) *Publisher {
 }
 
 func (p *Publisher) NewStreamer(kvStore kv.KeyValueStore) (*Streamer, error) {
+	intDb, err := kvStore.GetInternalDatabase()
+	if err != nil {
+		ulog.E(err)
```

Comment:
No return on error, is it intentional?

If not, it better be:

```
	if ulog.E(err) {
		return nil, err
    }
```

Reply:
Great catch, I was focusing on adding err, but it's always nil for `GetInternalDatabase()`. I changed this.